src/bin/add/args.rs

```diff
@@ -85,17 +85,11 @@ pub struct Args {
     pub optional: bool,
 
     /// Path to the manifest to add a dependency to.
-    #[structopt(long = "manifest-path", value_name = "path", conflicts_with = "pkgid")]
-    pub manifest_path: Option<PathBuf>,
+    #[structopt(flatten)]
+    pub manifest: clap_cargo::Manifest,
 
-    /// Package id of the crate to add this dependency to.
-    #[structopt(
-        long = "package",
-        short = "p",
-        value_name = "pkgid",
-        conflicts_with = "path"
```

Comment:
Hmm, this is dropping the confict.  We'll have to double check that

Reply:
nm.  `path` doesn't refer to anything.  it should refer to `manifest_path` but that conflict shouldn't exist